src/F0_output.py

```diff
@@ -134,44 +175,46 @@ def store_timeseries_all_busses_to_excel(dict_values):
     return
 
 
-def store_as_json(dict_values, file_name):
+def convert(o):
     """
-
-    :param dict_values:
-    :param file_name:
-    :return:
+    This converts all data stored in dict_values that is not compatible with the json format to a format that is compatible.
+    :param o: Any type. Object to be converted to json-storable value.
+    :return: json-storable value.
     """
+    if isinstance(o, numpy.int64):
+        return int(o)
```

Comment:
In general it is easier for debugging, that a function contains only one `return` statement (you can define a variable `answer` which you only at the end (potentially after running some checks on it) send to the `return answer`

Reply:
Makes sense.